src/components/ReportActionItem/ReportPreview.tsx

```diff
@@ -151,8 +152,9 @@ function ReportPreview({
         // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
         formattedMerchant ||
         translate('iou.requestCount', {
-            count: numberOfRequests - numberOfScanningReceipts,
+            count: numberOfRequests - numberOfScanningReceipts - numberOfPendingRequests,
```

Comment:
Hope there will be no case of this value being negative.
`reportPreviewAction.childMoneyRequestCount` - numberOfScanningReceipts - numberOfPendingRequests